packages/pubsub/src/subscription.js

```diff
@@ -520,6 +523,9 @@ Subscription.prototype.exists = function(callback) {
 Subscription.prototype.flushQueues_ = function() {
   var self = this;
 
+  clearTimeout(this.flushTimeoutHandle_);
```

Comment:
I don't think this and the next line is tested.

Reply:
Ah! Yeah, good catch.